### ISSUE TITLE
wsj

### ISSUE THREAD

**Issue by hubery98** [magnolia1234/bypass-paywalls-chrome-clean#59]
Hello, bro. The site WSJ can't read the full story. Please fix it.

**Comment by magnolia1234** [comment]
Which platform and browser?
For me, only mobile Yandex isn't working on WSJ.

**Comment by suhuaixing** [comment]
me too

Google Chrome | 80.0.3987.149 (Official Build) (64-bit)
-- | --
Revision | 5f4eb224680e5d7dca88504586e9fd951840cac6-refs/branch-heads/3987_137@{#16}
OS | Linux
JavaScript | V8 8.0.426.27
Flash | 32.0.0.344 /home/huaixing/.config/google-chrome/PepperFlash/32.0.0.344/libpepflashplayer.so
User Agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36

**Comment by magnolia1234** [comment]
> me too (Linux/Chrome)

You're not using an user-agent/referer extension or a proxy-server?
WSJ needs Googlebot-fix.

**Comment by suhuaixing** [comment]
I'm using a proxy-server and I have to, since I'm in China.
Linux and Windows both get tested:
Chrome is not working
but firefox 74[1] is working

This began from today. It's working fine before. 

I tried incognito mode and disabled all plugins except this one.

first three or four articles are fine, about the fourth, the paywall shows.
I closed the  incognito mode and reopen a new one, the same hapened, it seems wsj use coockies.

So, I blocked the coockies manually, hah, It's working. I can read all articles again!

It seems the plugin doesn't block the wsj cookies anymore since today, maybe wsj updates something?

![Screenshot_20200320_015752](https://user-images.githubusercontent.com/42117215/77099250-98b1bd80-6a4e-11ea-9ede-984196891f80.png)

[1] https://github.com/magnolia1234/bypass-paywalls-firefox-clean
